theHarvester/__main__.py

```diff
@@ -125,7 +125,7 @@ def store(search_engine: Any, source: str, process_param: Any = None, store_host
                 for usr in sorted(list(set(people_list))):
                     print(usr)
         if store_links:
-            links = linkedin_links_search.get_links()
+            links = search_engine.get_links()
             db.store_all(word, links, 'name', 'linkedin')
             if len(links) == 0:
                 print('\n[*] No links found Linkedin.\n\n')
```

Comment:
Replace linkedin with the engine name that is searching for links

Reply:
Thanks for spotting that!